src/lib/containers/entity_finder/tree/TreeView.tsx

```diff
@@ -98,6 +125,19 @@ export const TreeView: React.FunctionComponent<TreeViewProps> = ({
     },
   )
 
+  const {
+    data: currentContainerBundle,
+    isSuccess: isSuccessBundle,
+  } = useGetEntityBundle(
+    sessionToken,
+    currentContainer!,
+    BUNDLE_REQUEST_OBJECT,
+    undefined,
+    {
+      enabled: !!currentContainer && currentContainer !== 'root',
+    },
+  )
```

Comment:
We get the entity bundle mostly for the path, to show breadcrumbs. We get the bundle because in most cases, it will probably already be in the cache.